lib/neoboard/widgets/mattermost.ex

```diff
@@ -0,0 +1,44 @@
+defmodule Neoboard.Widgets.Mattermost do
+  use GenServer
+  use Neoboard.Pusher
+  use Neoboard.Config
+  alias Neoboard.TimeService
+  alias Neoboard.Widgets.Mattermost.Fetcher
+
+  def start_link do
+    {:ok, pid} = GenServer.start_link(__MODULE__, :ok)
+    send(pid, :tick)
+    :timer.send_interval(config[:every], pid, :tick)
+    {:ok, pid}
+  end
+
+  def init(:ok) do
+    HTTPoison.start
```

Comment:
I *think* `HTTPoison.start` not need here because it will be started as application defined via `mix.exs`.